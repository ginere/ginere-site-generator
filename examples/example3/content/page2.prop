PAGE_NAME=Page 2
PAGE_CONTENT={page2-content.html}
template=template.html