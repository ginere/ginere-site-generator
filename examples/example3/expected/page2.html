<!DOCTYPE html>
<html>
  <head>
  <meta name="description" content="The site description">
  <title>Page 2</title>
</head>
  <body>	
	Content of the page 2	
	</p>
The common footer
  </body>
</html>
